app/javascript/packs/articleModerationTools.js

```diff
@@ -1,28 +1,30 @@
 /* global userData */
 import { isModerationPage } from '@utilities/moderation';
 
-const user = userData();
-const { authorId: articleAuthorId, path } = document.getElementById(
-  'article-show-container',
-).dataset;
+getCsrfToken().then(() => {
```

Comment:
Added to make sure userdata is always available by the time we start this process